### PR TITLE
Help tabs

### PR DESCRIPTION
**Pull request by jamestewartjr**
Hey @jrob8577 @tjfwalker here is some WIP.
what's missing:
-routes for connecting 66 & 67
-styling
-getting the viewed indicator toggling.
-popping up help first after signup.

### CODE REVIEW

File: public/stylesheets/style.css [hunk 1]
```diff
@@ -21,3 +21,15 @@
   text-decoration: line-through;
   color: #777;
 }
+
+.button {
```
Comment (jrob8577) [inline public/stylesheets/style.css]: We should explore the use of existing bootstrap button classes so that our styles remain internally consistent.

**Comment by jrob8577** [comment]
Closing; incomplete feature

**Comment by tjfwalker** [comment]
You'd told us to do PR for incomplete features so that we could get feedback. Right?

File: public/stylesheets/style.css (hunk 1 above)
Comment (tjfwalker) [inline public/stylesheets/style.css]: Are we not waiting on a style guide?

**Comment by jrob8577** [comment]
@tjfwalker There's not much functionality here to leave feedback on.  HTML/CSS looks reasonable.  In terms of style guide, we're using Bootstrap until the style guide is official.

**Pull request closed without merging**